src/test/java/at/ac/tuwien/kr/alpha/solver/HanoiTowerTest.java

```diff
@@ -82,12 +90,13 @@ private void testHanoiTower(int instance) throws IOException {
 	}
 
 	private void testHanoiTower(String instance) throws IOException {
+		ignoreTestForNaiveSolver();
```

Comment:
Please move the conditional-ignore to the level of the unit test and not nested inside some (private) testing code, i.e., make the call to `ignoreTestForNaiveSolver();` at the beginning of `testSimple()`.